keps/1282-requeue-strategy/README.md

```diff
@@ -0,0 +1,306 @@
+# KEP-1282: Requeue Strategy
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [API Changes](#api-changes)
+  - [Changes to Queue Sorting](#changes-to-queue-sorting)
+    - [Existing Sorting](#existing-sorting)
+    - [Proposed Sorting](#proposed-sorting)
+    - [Preemption/Preemptor Flapping](#preemptionpreemptor-flapping)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+Introduce new options that allow administrators to configure how Workloads are placed back in the queue after being after being evicted.
+
+## Motivation
+
+### Goals
+
+Allow administrators to configure requeuing behavior based on the reason the Workload is being requeued.
+
+### Non-Goals
+
+## Proposal
+
+Make queue placement after preemption configurable at the level of Kueue configuration.
+
+### User Stories (Optional)
+
+#### Story 1
```

Comment:
We only have one story here, whereas the proposal covers both scenarios of preemption and timeout eviction. Should we add another story here, or maybe we can narrow down the scope of the KEP to only `waitForPodsReady`? I would suggest the second approach if possible.

Reply:
+1, if you plan to make the requeueing for other eviction mechanisms configurable, it needs to solve a valid use case. Otherwise, we can just focus on waitForPodsReady and leave other evictions as non-goals.